package.json

```diff
@@ -180,7 +180,7 @@
     "abi-decoder": "^2.4.0",
     "axios": "0.21.1",
     "bignumber.js": "9.0.1",
-    "bnc-onboard": "~1.30.0",
+    "bnc-onboard": "git+https://github.com/KeystoneHQ/onboard.git",
```

Comment:
why do we have to use this instead of the official onboard build? Wallet integrations are a critical part of our application and we rely on blocknative to deliver important updates for different wallets

Reply:
I should've read the description first 🤦

Reply:
Hi @mikheevm , thanks for your review.
This is because we fixed several bugs in keystone wallet in onboard, which is blocking our integration. So we forked bnc-onbard v1.33.0 and patched fix on it, using github link to make the change workable. 
We have raised PR to onboard: https://github.com/blocknative/onboard/pull/656 and wait for their review. 
We can keep this PR open until onboard's PR merged and change the version to the official build. Before that, we can use this PR to test the functionality. 😃